frontend/src/components/UserTables/UnscheduledTable.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Passenger } from '../../types/index';
+import './unscheduledTable.css';
+import RiderCard from '../RiderCard/RiderCard';
+
+
+function renderTableHeader() {
+  return (
+    <tr>
+      <th></th>
+      <th className="tableHeader">Time</th>
+      <th className="tableHeader">Passenger</th>
+      <th className="tableHeader">Pickup Location</th>
+      <th className="tableHeader">Dropoff Location</th>
+      <th className="tableHeader">Needs</th>
+    </tr>
+  );
+}
+
+const Table = () => {
+  const passengers = [
+    { startTime: '8:20am', endTime: '8:40am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+    { startTime: '8:30am', endTime: '8:50am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+    { startTime: '9:10am', endTime: '9:40am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+    { startTime: '9:30am', endTime: '9:50am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+    { startTime: '10:10am', endTime: '10:30am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+
+  ];
+
+  function renderTableData(allPassengers: Passenger[]) {
+    let currentTime = '';
+    return allPassengers.map((rider, index) => {
+      const { startTime, endTime, name, pickupLocation, pickupTag,
+        dropoffLocation, dropoffTag, needs } = rider;
+      const colon = startTime.indexOf(':');
+      const timeOfDay = startTime.substring(startTime.indexOf('m') - 1).toUpperCase();
+      const startHour = startTime.substring(0, colon) + timeOfDay;
+      if (startHour !== currentTime) {
+        currentTime = startHour;
+      } else {
+        currentTime = '';
+      }
+
+      const timeframe = currentTime;
+      const comma = ',';
+      const pickup = pickupLocation + comma + pickupTag;
+      const dropoff = dropoffLocation + comma + dropoffTag;
+      const inputValues = [name, pickup, dropoff, needs];
+      const tags = [1, 2];
+      return (
+        <tr key={index}>
+          <td className="cell">{timeframe}</td>
+          <td className="cell"><span style={{ fontWeight: 'bold' }}>{startTime}</span> <br></br> <span style={{ color: '#707070' }}>-- {endTime}</span></td>
+          <RiderCard values={inputValues} tags={tags} />
```

Comment:
since youre using the rider card for the unscheduled rides table, i think giving it a more general name would be better. also i think you could create a specific type for values, something like
```
type value = {
  data: string
  tag?: string
  // other fields if necessary
}
```
i think that would make displaying the table values easier instead of having to check and process multiple things